pkg/controller/pod_driver.go

```diff
@@ -705,3 +724,48 @@ func (p *PodDriver) OverwirteMountPodResourcesWithPVC(ctx context.Context, pod *
 	pod.Spec.Containers[0].Resources = resources
 	return nil
 }
+
+// checkingMountPodStuck check mount pod is stuck or not
+// maybe fuse deadlock issue, they symptoms are:
+//
+// 1. pod in terminating state
+// 2. after 1 minute, the pod is still alive.
+//
+// if those conditions are true, we need to manually tear down the fuse
+// connection so the pod doesn't get stuck.
+// we can do this to abort fuse connection:
+//
+//	echo 1 >> /sys/fs/fuse/connections/$dev_minor/abort
+func (p *PodDriver) checkingMountPodStuck(pod *corev1.Pod) {
+	if pod == nil || getPodStatus(pod) != podDeleted {
+		return
+	}
+
+	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Minute)
+	defer cancel()
+	for {
+		select {
+		case <-ctx.Done():
+			goto abort
+		default:
+			_, err := p.Client.GetPod(ctx, pod.Name, pod.Namespace)
+			if apierrors.IsNotFound(err) {
+				return
+			}
+			time.Sleep(10 * time.Second)
+		}
+	}
+
+abort:
+	if devMinor, ok := pod.Annotations[config.MountPointDevMinorKey]; ok {
+		err := os.WriteFile(fmt.Sprintf("/sys/fs/fuse/connections/%s/abort", devMinor), []byte("1"), 0600)
+		if err == nil {
+			return
+		}
+		klog.Warningf(`
+		mountpod %s/%s may be stuck in terminating state, you may need manually tear down the fuse connections in node %s.
```

Comment:
既然 csi-node abort 失败，其实提示用户手动去 echo 应该也帮助不大了，把写文件具体 err 打出来吧，方便后续排查。主要我们还没遇到过 echo abort 失败的情况
```suggestion
		cannot abort FUSE connection /sys/fs/fuse/connections/%s/abort for mount pod %s/%s, error %v.
```

Reply:
这里可能的原因是 fuse 目录不存在。比如升级后但是没有在 charts mount fuse volumes

Reply:
所以提示是有必要的，比如去排查问题的时候，可以直接从 node log 中获取需要 abort 的 connection？